detectable/src/main/java/com/synopsys/integration/detectable/detectable/executable/impl/SimpleExecutableResolver.java

```diff
@@ -92,6 +93,9 @@ public File resolveBash() {
         return findCachedSystem("bash");
     }
 
+    @Override
+    public File resolveCargo() { return findCachedSystem("cargo"); }
```

Comment:
Can you make this multi line to be consistent with the rest of the file?

Reply:
<img width="1115" alt="Screen Shot 2020-05-08 at 3 54 09 PM" src="https://user-images.githubusercontent.com/51929980/81443855-3159e500-9144-11ea-9dd0-119c4c597110.png">
in Intellij all the methods with a one-line body are single-line

Reply:
https://stackoverflow.com/questions/35995083/intellij-collapsing-single-line-methods